packages/manager/modules/pci/src/components/project/images-list/images-list.service.js

```diff
@@ -38,9 +97,17 @@ export default class ImagesList {
   }
 
   getImages(serviceName) {
-    return this.OvhApiCloudProjectImage.v6()
-      .query({ serviceName })
-      .$promise.then((images) => ImagesList.groupByName(images));
+    return this.$http
+      .get(`/cloud/project/${serviceName}/image`)
+      .then(({ data: images }) => {
+        images.forEach((image) => {
+          Object.assign(image, {
```

Comment:
instead of Object.assign it's preferred to use
```js
return {
  ...image,
  nameGeneric: ...,
  distribution: ...,
};
```

Reply:
I do like this, i will recreate the whole images list.
In my case i need just to add two new attribute ( `nameGeneric` and `distribution`).

Reply:
Maybe like this:
```suggestion
          image.nameGeneric = '...'
          image.distribution = '...'
```

Like this we avoid the Object.assign, is't ok @frenautvh ?

Reply:
This looks achievable by a `map` 
Also as @frenautvh mentionned, spread operator is preffered over `Object.assign` as it prevents updating the value and creates a new object instead